contracts/Liquidator/Liquidator.sol

```diff
@@ -147,18 +162,24 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Acce
     /// @notice Thrown if trying to liquidate any token when VAI debt is too high
     error VAIDebtTooHigh(uint256 vaiDebt, uint256 minLiquidatableVAI);
 
+    /// @notice Thrown when transfer of underlying assets to protocol share reserve failed
+    error UnderlyingTransferFailed(address vToken, address underlying);
+
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
     /// @notice Constructor for the implementation contract. Sets immutable variables.
     /// @param comptroller_ The address of the Comptroller contract
     /// @param vBnb_ The address of the VBNB
     /// @param treasury_ The address of Venus treasury
+    /// @param wBNB_ The address of wBNB
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor(address comptroller_, address payable vBnb_, address treasury_) {
+    constructor(address comptroller_, address payable vBnb_, address treasury_, address wBNB_) {
```

Comment:
Shouldn't we include the `wBNB_` in the `initialize` function instead to allow us to set it up during the re-initialization?

Reply:
Instead of setting it at re-initialization, why not do it during deployment and keep it immutable like the vBNB variable Wdys?